src/CheckoutController.php

```diff
@@ -2,7 +2,9 @@
 
 namespace Maxfactor\Checkout;
 
+use Session;
```

Comment:
Please can you use the full Facade namespace here?

Reply:
Sorry, no probs!